src/platforms/javascript/guides/astro/manual-setup.mdx

```diff
@@ -0,0 +1,254 @@
+
+If you add a `sentry.client.config.js` or `sentry.server.config.js` file, the options specified in the [Sentry Astro integration](#astro-integration-options) are ignored for the respective runtime.
+The only exception is `sourceMapsUploadOptions` which **always** needs to be set in the integration options.
+
+</Alert>
+
+### Client Side
+
+Here's an example of a custom client-side SDK setup:
+
+```javascript {filename:sentry.client.config.js}
+import * as sentry from "@sentry/astro";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  replaysSessionSampleRate: 0.1,
+  replaysOnErrorSampleRate: 1.0,
+  integrations: [
+    new Sentry.Replay({
+      maskAllText: true,
+      blockAllMedia: true,
+    }),
+    new BrowserTracing({
+      tracePropagationTargets: [/\//, "my-api-domain.com"],
+    }),
+  ],
+  tracesSampler: (samplingContext) => {
+    if (samplingContext.transactionContext.name === "/home") {
+      return 0.5;
+    }
+    return 0.1;
+  },
+});
+```
+
+### Server Side
+
+Here's an example of a custom server-side SDK setup:
+
+```javascript {filename:sentry.server.config.js}
+import * as sentry from "@sentry/astro";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  tracesSampler: (samplingContext) => {
+    if (samplingContext.transactionContext.name === "/home") {
+      return 0.5;
+    }
+    return 0.1;
+  },
+  beforeSend: (event) => {
+    console.log("before send", event);
+    return event;
+  },
+});
+```
+
+### Changing config files location
+
+If you want to change the location of your `sentry.(client|server).config.js` files, specify the path to the files in the Sentry Astro integration options in your `astro.config.mjs`:
+
+```javascript {filename:astro.config.mjs}
+export default defineConfig({
+  // Rest of your Astro project config
+  integrations: [
+    sentryAstro({
+      clientInitPath: ".config/sentryClientInit.js",
+      serverInitPath: ".config/sentryServerInit.js",
+    }),
+  ],
+});
+```
+
+## Configure Source Maps Upload
+
+You can specify a few options to configure source maps upload for your production builds. Generally, this should work if you followed the [Astro CLI installation guide](../#add-readable-stack-traces-to-errors).
+
+Note that source maps upload only works if an auth token is specified.
+
+### Working with old authentication tokens
+
+If you're not using our new [organization-scoped auth tokens](/product/accounts/auth-tokens/#organization-auth-tokens) yet, you need to additionally specify your `org` slug in your `sourceMapsUploadOptions`:
+
+```javascript {filename:astro.config.mjs}
+export default defineConfig({
+  integrations: [
+    sentryAstro({
+      // rest of your sentry options
+      sourceMapsUploadOptions: {
+        project: "___PROJECT_SLUG___",
+        org: "___ORG_SLUG___",
+        authToken: process.env.SENTRY_AUTH_TOKEN,
+      },
+    }),
+  ],
+});
+```
+
+### Disable Source Maps Upload
+
+You can disable automatic source maps upload in your Vite config:
```

Comment:
Wasn't sure if this was just a copy/paste typo? Or does Astro refer to this file as your Vite config?

```suggestion
You can disable automatic source maps upload in your Astro config:
```

Reply:
```suggestion
You can disable automatic source maps upload in Astro's Vite config:
```

Reply:
Copy/paste, Astro config is correct. Good catch, thanks!